build.sbt

```diff
@@ -235,8 +235,9 @@ lazy val finch = project.in(file("."))
   .settings(libraryDependencies ++= Seq(
     "io.circe" %% "circe-generic" % circeVersion
   ))
+  // Bring back examples when Twitter Server is published for Scala 2.13.
   .aggregate(
-    core, fs2, iteratee, generic, argonaut, circe, benchmarks, test, jsonTest, examples, refined
+    core, fs2, iteratee, generic, argonaut, circe, benchmarks, test, jsonTest, /*examples,*/ refined
```

Comment:
Doesn't `examples` work just fine with `crossScalaVersions := Seq("2.12.7")` setting?

Reply:
It works fine for `sbt validate` which doesn't cross build, but breaks on things like `sbt +compile` trying to fetch 2.13 dependencies. I _think_ its because its inheriting the crossVersions of the aggregate project and not respecting the child projects override.

Reply:
Yeah, I see it now:

```
 Issuing a cross building command, but not all sub projects have the same cross build configuration. This could result in subprojects cross building against Scala versions that they are not compatible with. Try issuing cross building command with tasks instead, since sbt will be able to ensure that cross building is only done using configured project and Scala version combinations that are configured.
```

I wish I'd understand whatever the `Try issuing cross building command with tasks instead` means